File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
 // Ask for the pyramids' height
int height;
do
{
    height = get_int("Height of the pyramids is: ");
}
while (height < 1 || height > 8);

 // Print the pyramids
int row;
for (row = 1; row <= height; row++)
{
    for (int i = 0; i < height - row; i++)
    {
        printf(" ");
    }
    for (int j = 0; j < row; j++)
    {
        printf("#");
    }
    printf("  ");
    for (int h = 0; h < row; h++)
    {
        printf("#");
    }
    printf("\n");
}
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
height = get_int("Height: ")
while height < 0:
    height = get_int("Height ")
for row in range(1, height + 1, 1):
    for spaces in range(height - row):
        print(" ", end = "")
    for stars in range(row):
        print("#", end = "")
    print ("  ",end = "")
    for stars in range(row):
        print("#")